### ISSUE TITLE
libpythonscript.dylib linked to libpython3.6m.dylib in travis directory

### ISSUE THREAD

**Issue by ColinKinloch** [touilleMan/godot-python#76]
In godot-python installed from the AssetLib, libpythonscript.dylib is linked to "/Users/travis/build/touilleMan/godot-python/platforms/osx-64/cpython_build/lib/libpython3.6m.dylib"

As you'd expect it works fine if "pythonscript/osx-64-cpython/lib" is symlinked to "/Users/travis/build/touilleMan/godot-python/platforms/osx-64/cpython_build/lib"

Godot Error:
```
0:00:00:0274 - Can't open dynamic library: /Users/ckinloch/godot_projects/python_test/pythonscript/osx-64-cpython/libpythonscript.dylib. Error: dlopen(/Users/ckinloch/godot_projects/python_test/pythonscript/osx-64-cpython/libpythonscript.dylib, 2): Library not loaded: /Users/travis/build/touilleMan/godot-python/platforms/osx-64/cpython_build/lib/libpython3.6m.dylib
  Referenced from: /Users/ckinloch/godot_projects/python_test/pythonscript/osx-64-cpython/libpythonscript.dylib
  Reason: image not found
----------
Type:Error
Description: Can't open dynamic library: /Users/ckinloch/godot_projects/python_test/pythonscript/osx-64-cpython/libpythonscript.dylib. Error: dlopen(/Users/ckinloch/godot_projects/python_test/pythonscript/osx-64-cpython/libpythonscript.dylib, 2): Library not loaded: /Users/travis/build/touilleMan/godot-python/platforms/osx-64/cpython_build/lib/libpython3.6m.dylib
  Referenced from: /Users/ckinloch/godot_projects/python_test/pythonscript/osx-64-cpython/libpythonscript.dylib
  Reason: image not found
Time: 0:00:00:0274
C Error: Method/Function Failed, returning: ERR_CANT_OPEN
C Source: platform/osx/os_osx.mm:1308
C Function: open_dynamic_library
```

otool -L pythonscript/osx-64-cpython/libpythonscript.dylib:
```
pythonscript/osx-64-cpython/libpythonscript.dylib:
	pythonscript/libpythonscript.dylib (compatibility version 0.0.0, current version 0.0.0)
	/Users/travis/build/touilleMan/godot-python/platforms/osx-64/cpython_build/lib/libpython3.6m.dylib (compatibility version 3.6.0, current version 3.6.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1238.60.2)
```

**Comment by touilleMan** [comment]
Thanks for the report !

Unfortunately I don't have experience with macOS ecosystem (and don't have a macbook) so do you think you could investigate more on this ?

It seems a bit strange given `platforms/osx-64/SCsub`  has been created from  `platforms/x11-64/SCsub` and the two are really similar (no absolute path appear there obviously)...

**Comment by touilleMan** [comment]
should be fixed by #77
